recipes-kernel/linux/linux-base_git.bbappend

```diff
@@ -9,4 +9,13 @@ CVE_VERSION = "${LINUX_CVE_VERSION}"
 # CVE-2021-26934: It's Xen document problem. Not a kernel bug. http://xenbits.xen.org/xsa/advisory-363.html
 # CVE-2021-43057: This issue was introduced in 5.13-rc1. 4.19.y is not affected.
 # CVE-2022-29582: This is io_uring issue. linux 4.19 doesn't have this feature.
-CVE_CHECK_WHITELIST = "CVE-2021-26934 CVE-2021-43057 CVE-2022-29582"
+# CVE-2021-42327: This issue was introduced in 5.10-rc1. 4.19.y is not affected.
+# CVE-2021-45402: This issue was introduced in 5.7. 4.19.y is not affected.
+# CVE-2022-0168: This issue was introduced in 4.20-rc1. 4.19.y is not affected.
+# CVE-2022-1508: This is io_uring issue. linux 4.19 doesn't have this feature.
+# CVE-2022-1789: EMLinux 2.x doesn't support x86 architecture.
```

Comment:
This reason cannot be accepted for the whitelist.
But this CVE is not affected to 4.19.y, so you should write "This issue was introduced in <version>. 4.19.y is not affected." here.

Reply:
@hiraku-wfs thanks. updated.